### PR TITLE
Add support for creating sensor entities in HomeAssistant

### PR DESCRIPTION
**Pull request by insertjokehere**
This uses the MQTT Discovery component in HomeAssistant to create
sensors for each property of each Mi Flora device

See https://home-assistant.io/docs/mqtt/discovery/ for details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The program can be executed in **daemon mode** to run continuously in the backgr
     * JSON encoded
     * following the [Homie Convention v2.0.5](https://github.com/marvinroger/homie)
     * following the [mqtt-smarthome architecture proposal](https://github.com/mqtt-smarthome/mqtt-smarthome)
+	* using the [HomeAssistant MQTT discovery format](https://home-assistant.io/docs/mqtt/discovery/)
```
Comment (ThomDietrich) [inline README.md]: Please replace tab by four spaces :)

**Comment by ThomDietrich** [comment]
Nice addition. Just one minor request.

The script is quickly growing out of its comfortable size for a mostly sequential script 🙈
Btw. the imho next step in development would be to update the used miflora library and switch over to the bluepy backend: https://github.com/open-homeautomation/miflora#backends
I'm currently busy, if you want to take a hit... ;)

**Comment by insertjokehere** [comment]
Heh, will have a look if I feel like my TODO list is getting too short...

Whats the advantage in switching to BluePy?

**Comment by ThomDietrich** [comment]
The gatttools backend works by calling the actual gatttools command from within python: https://github.com/open-homeautomation/miflora/blob/master/miflora/backends/gatttool.py#L56
Nasty stuff!
In addition bluez/gatttools needs to be installed which is tricky on some platforms, bluepy is a library that would be installed via the `pip install -r requirements.txt` step.

**Pull request merged**